src/util/spreadsheet/misc.py

```diff
@@ -1,10 +1,13 @@
-from .. import json
+from .. import json, pg
+
+BATCH_SIZE = 10
```

Comment:
```suggestion
BATCH_SIZE = 10
```

Have you tried larger values too? Is there any impact on the performance?

Reply:
I did, it takes roughly the same time. Since some records holds up to ~13Mo, I thought a 100 records could already be too much.

Reply:
> In fact I was not sure how much we could gain by copying those tuples into memory in batches

It' less about memory copy, it's about network round trips. Would we expect other DBs to potentially have a lot of small tuples instead of a few big ones in this table? If so, then it may be worth it to be clever about it, like selecting the max-sized commands and from that determine an itersize.

Reply:
Something like this?
` itersize = int(available_memory / biggest_command)`
How can I determine `available_memory` ?